TwilioStore.Tests/Services/NotificationServiceTest.cs

```diff
@@ -0,0 +1,89 @@
+﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Moq;
+using TwilioStore.Interfaces.Exceptions;
+using TwilioStore.Interfaces.Services;
+using TwilioStore.Services;
+
+namespace TwilioStore.Tests.Services
+{
+    [TestClass]
+    public class NotificationServiceTest
+    {
+        // Get your test credentials from https://www.twilio.com/console/account/settings
+        private const string TestAccountSid = "ACxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx";
+        private const string TestAuthToken = "0123456789abcdef0123456789abcdef";
+
+        // Magic numbers from https://www.twilio.com/docs/api/rest/test-credentials
+        private const string ValidFromNumber = "+15005550006";
+        private const string ValidToNumber = "+18778894546";
+        private const string InvalidNumber = "+15005550001";
+        private const string AvailableNumber = "+15005550006";
+        private const string UnavailableNumber = "+15005550000";
+
+        private static INotificationConfiguration GetTestConfig()
```

Comment:
That would certainly work. However, I don't like being tightly coupled to System.Configuration. If you wanted to run two different tests that used different configuration values, then using app.config wouldn't work.

You can also manually assign values to the AppSettings at runtime (they're not read-only, interestingly enough), but, again, this can cause problems if you have tests that you want to have use different configuration values and the tests are run in parallel.